musicbrainz-server/recipes/production.rb

```diff
@@ -15,7 +15,17 @@
 daemontools_service "musicbrainz-server" do
   directory "/home/musicbrainz/svc-musicbrainz-server"
   template "musicbrainz-server"
-  variables :nproc => node['musicbrainz-server']['nproc']
+  variables :nproc => node['musicbrainz-server']['nproc-website']
+  action [:enable,:start]
+  subscribes :restart, "git[/home/musicbrainz/musicbrainz-server]"
+  subscribes :restart, "template[/home/musicbrainz/musicbrainz-server/lib/DBDefs.pm]"
+  log true
+end
+
+daemontools_service "musicbrainz-ws" do
+  directory "/home/musicbrainz/svc-musicbrainz-ws"
+  template "musicbrainz-server"
```

Comment:
I do think this needs changing, to 'musicbrainz-ws', so it finds the right templates. Unless, of course, you just want to have one set and change the socket to be a configurable variable, which I think would work too?